src/app/models/types/pageDataParams.ts

```diff
@@ -1,11 +1,11 @@
 import { ParsedUrlQuery } from 'querystring';
-import { PageTypes, Services, Variants } from './global';
+import { PageTypes, Services } from './global';
 
 export default interface PageDataParams extends ParsedUrlQuery {
   id: string;
   page?: string;
   service: Services;
-  variant?: Variants;
+  variant?: string[];
```

Comment:
This is technically correct even though it looks like the wrong change. Params come through as an array of strings from the Next.js catch-all routes `[[...variant]]`. The name `variant` there doesn't mean anything, its just a name for the file, it could equally be called `[[...]]` and it would work the same.